.github/workflows/checks.yml

```diff
@@ -22,13 +22,6 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
-      - name: Get Terraform CLI Version
-        id: vars
-        run: echo ::set-output name=tf-version::$(cat .terraform-version)
-      - name: Setup Terraform
-        uses: hashicorp/setup-terraform@v2
-        with:
-          terraform_version: ${{ steps.vars.outputs.tf-version }}
```

Comment:
I think there may too more of these to remove, just search "Get Terraform CLI Version".

Reply:
@stevenelleman-okta We just don't need it for the unit tests since those are just directly on the provider Go code. We do need it for the acceptance tests still which are the other two entries.